tests/virt/node/hotplug/test_cpu_memory_hotplug.py

```diff
@@ -88,8 +88,9 @@ class TestCPUHotPlug:
         "hotplugged_sockets_memory_guest", [pytest.param({"sockets": SIX_CPU_SOCKETS})], indirect=True
     )
     @pytest.mark.dependency(name=f"{TESTS_CLASS_NAME}::hotplug_cpu")
+    @pytest.mark.special_infra
```

Comment:
this one should go as test param (only windows tests need special_infra, rhel should be able to run on any cluster)